hack/build-local-images.py

```diff
@@ -138,12 +149,11 @@ def image_rebuild_requested(image):
     suffix explicitly or does not provide
     any explicit requests.
     """
-    return len(sys.argv) == 1 or (
-        len(sys.argv) > 1 and (
-            image in sys.argv or
-            full_name(image) in sys.argv
-        )
-    )
+    build_image = image_config[image].get("enable_default", True)
+    if len(sys.argv) == 1:
+        return build_image
+    if image in sys.argv or full_name(image) in sys.argv:
```

Comment:
Why did we drop the `len(sys.argv) > 1` check here?

Reply:
Because the return above makes the check at that point unnecessary. Any additional argument makes argv not equal to one.